routing/base/astar_algorithm.hpp

```diff
@@ -315,16 +323,22 @@ template <typename TGraph>
 typename AStarAlgorithm<TGraph>::Result AStarAlgorithm<TGraph>::FindPath(
     TGraphType & graph, TVertexType const & startVertex, TVertexType const & finalVertex,
     RoutingResult<TVertexType, TWeightType> & result, my::Cancellable const & cancellable,
-    TOnVisitedVertexCallback onVisitedVertexCallback) const
+    OnVisitedVertexCallback onVisitedVertexCallback, CheckWeightCallback checkWeightCallback) const
 {
   result.Clear();
   if (nullptr == onVisitedVertexCallback)
```

Comment:
Could you please fix this Yoda-condition?

Reply:
done